cost/superseded_instance/README.md

```diff
@@ -0,0 +1,35 @@
+## Superseded Instances Policy Template
+
+### What it does
+
+This Policy Template uses data in [Policies Data](https://github.com/rightscale/policy_templates/tree/master/data) to determine if an instance type has been superseded, it will then list all the instances that have been superseded and their types. 
```

Comment:
Should explain the difference with this policy over optima,.  I don't think a user cares about the Policy Data.  Maybe that part should go into the Usage area.